app/controllers/v2/content_items_controller.rb

```diff
@@ -1,5 +1,11 @@
 module V2
   class ContentItemsController < ApplicationController
+    def index
+      format = params.fetch(:format)
```

Comment:
I've been back and forth on this. I've changed it all to `content_format` again to avoid any confusion.